incubator/kiam/README.md

```diff
@@ -0,0 +1,139 @@
+# kiam
+
+Installs [kiam](https://github.com/uswitch/kiam) to integrate AWS IAM with Kubernetes.
+
+## TL;DR;
+
+```console
+$ helm install incubator/kiam
+```
+
+## Introduction
+
+This chart bootstraps a [kiam](https://github.com/uswitch/kiam) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.8+ with Beta APIs enabled
+
+## Installing the Chart
+
+In order for the chart to configure kiam correctly during the installation process you should have created and installed TLS certificates and private keys as described [here](https://github.com/uswitch/kiam/blob/master/docs/TLS.md).
+
+> **Tip**: The `hosts` field in the kiam server certificate should include the value _release-name_-server:_server-service-port_, e.g. `my-release-server:443`
```

Comment:
@ewbankkit I was banging my head against the wall for a few hours trying to figure an error I was seeing in the agent logs, which boiled down to the hosts in the server cert. I would like to suggest the following change:

> **Tip**: The `hosts` field in the kiam server certificate should include the value `<release-name>-kiam-server:<server-service-port>`, e.g. `my-release-kiam-server:443`
>
> If you don't include the exact hostname used by the kiam agent to connect to the server, you'll see a warning (which is really an error) in the agent logs similar to the following, and your pods will fail to obtain credentials:
>
> `{"level":"warning","msg":"error finding role for pod: rpc error: code = Unavailable desc = there is no connection available","pod.ip":"100.120.0.2","time":"2018-05-24T04:11:25Z"}`

(Note the added `kiam-` part added after `my-release` and before `server`)

Reply:
@luispollo Thanks, I'll add that.